hack/tools/install_kubebuilder.sh

```diff
@@ -4,16 +4,15 @@
 
 source ./common.sh
 
-version=2.3.1
 
-header_text "Checking for bin/etcd"
-[[ -f bin/etcd ]] && exit 0
+header_text "Checking for bin/etcd, bin/kube-apiserver and bin/kubectl ${KUBE_VERSION}"
```

Comment:
why do we install kubebuilder for arrow-flight module ?

Reply:
it contains the kubectl installation